webrtc.html

```diff
@@ -7405,18 +7405,18 @@ <h3>priority</h3>
      <section id="maxbitrateinsimulcast">
        <h3>maxBitrate</h3>
        <p><code><a>RTCRtpEncodingParameters</a></code> dictionary member <code>maxBitrate</code>
-         can be utilized to set the maximum bitrate of a simulcast stream. However,
-         the user agent is not required to take <code>maxBitrate</code> into account when
-         responding to congestion, such as modifying the allocation of bandwidth between
-         simulcast streams or deciding when to stop sending one or more streams.</p>
+         can be utilized to set the maximum bitrate of a simulcast stream. However, the user
+         agent may not take <code>maxBitrate</code> into account in responding to congestion,
```

Comment:
Still think "may not" is a bad echo of the RFC 2119 MAY NOT (equivalent of MAY); suggest "may choose to not" to break it up.